release/helpers/changelog-helper.mjs

```diff
@@ -5,15 +5,12 @@
  * @param issues {Array<{id: string, key: string, githubIssue: string, summary: string, components: Array<string>, issueTypeName: string, statusName: string}>}
  */
 export function getChangelogFor(title, issues) {
-  const authorizedIssueTypes = ['Public Bug', 'Public Security', 'Story'];
+  const authorizedIssueTypes = ['Public Bug', 'Public Security', 'Public Improvement', 'Story'];
 
   const filteredIssues = issues
     .filter((issue) => {
       return !!issue.issueTypeName && authorizedIssueTypes.includes(issue.issueTypeName);
     })
-    .filter((issue) => {
-      return issue.statusName === 'Done';
```

Comment:
Should we add more conditions instead of removing the line completely? The done status is interesting for us only.